src/constants/tokens.spec.ts

```diff
@@ -76,4 +76,25 @@ describe('constants/tokens', () => {
       });
     });
   });
+  describe('getOneTokens', () => {
+    type TestArgs = {
+      chainId: ChainId;
+    };
+
+    const testParams: TestArgs[] = [
+      {
+        chainId: ChainId.Mainnet
+      },
+      {
+        chainId: ChainId.Polygon
+      }
+    ];
+
+    testParams.forEach((testParam) => {
+      it(JSON.stringify(testParam), async () => {
+        const actualResult = getOneTokens(testParam.chainId);
+        expect(actualResult.length).toBeGreaterThan(0);
```

Comment:
Loop through each result and verify each one has isOneToken true, that's the true test here